account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -439,3 +442,29 @@ const requestPaymasterAndData: <C extends ClientWithAlchemyMethods>(
     };
   },
 });
+
+/**
+ * Middleware for managing gas fees with Alchemy's gas manager based on the provided configuration.
+ *
+ * @example
+ *  ```ts
+ * import { alchemyGasManagerMiddleware } from "@account-kit/infra";
+ *
+ * const client = createSmartAccountClient({
+ *      ...
```

Comment:
nit: at some point I'm going to add type checking to these examples and this will throw an error.

can you add the minimal required fields to this? they should be:
`transport` and `chain`. For transport just do `http("rpc-url")` and `chain` import `sepolia` from `account-kit/ifnra`. `http` is imported from `viem`

Reply:
Done